websites.yml

```diff
@@ -46,3 +46,40 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+avatar:
+  pinterest:
+    type: regex
+    property: image_xlarge_url
+    url: null
+  gitlab:
+    type: meta
+    property: og:image
+    url: null
+  github:
+    type: meta
+    property: og:image
+    url: null
+  tumblr:
+    type: meta
+    property: og:image
+    url: null
+  behance:
```

Comment:
Use `behance: false` , and order the keys alphabetically, so behance is at the top of the `avatar` group.